app/views/unattended/report_templates/duplicate_hosts.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+name: Duplicate hosts
+snippet: false
+template_inputs:
+- name: hosts
+  required: false
+  input_type: user
+  description: Limit the report only on hosts found by this search query. Keep empty
+    for report on all available hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+- name: attribute
```

Comment:
I think this is to general, I don't expect user to know the right attribute name. Also we know which attributes can indicate the duplicity. We may extend the list later, but I'd focus on
FQDN
IP
IPv6
MAC
UUID

for ips and mac, we also need to `:include => :primary_interface` to avoid N+1